roles/sft-server/molecule/default/verify.yml

```diff
@@ -0,0 +1,9 @@
+---
+# This is an example playbook to execute Ansible tests.
+
```

Comment:
copy&paste left over?

Reply:
PR coming up for this.

Reply:
https://github.com/wireapp/ansible-sft/pull/14